apps/mail/hooks/use-optimistic-actions.ts

```
@@ -13,6 +13,17 @@ import { useQueryState } from 'nuqs';
 import { useAtom } from 'jotai';
 import { toast } from 'sonner';
 
+type PendingAction = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PendingAction'. Consider to delete it or rename it.

'PendingAction' is defined here:


(lint/suspicious/noRedeclare)